src/base/utils/foreignapps.cpp

```diff
@@ -253,16 +265,17 @@ PythonInfo Utils::ForeignApps::pythonInfo()
         if (testPythonInstallation("python3", pyInfo))
             return pyInfo;
 #endif
-        // Look for "python" in Windows and in UNIX if "python3" is
-        // not detected.
-        if (testPythonInstallation("python", pyInfo))
```

Comment:
I would prefer to keep this (search in paths first). You can remove the `#if defined(Q_OS_UNIX)` conditional above (for finding python3).